compiler/optimizer/LocalAnalysis.cpp

```diff
@@ -714,7 +714,12 @@ bool TR_LocalAnalysisInfo::isCallLike(TR::Node *node) {
 
    if (node->getOpCode().hasSymbolReference())
       {
-      if (node->getSymbolReference()->getSymbol()->isVolatile() || node->getSymbolReference()->getSymbol()->isMethodMetaData())
+      if (node->getSymbolReference()->getSymbol()->isVolatile() || 
+          (node->getSymbolReference()->getSymbol()->isMethodMetaData()
+#if J9_PROJECT_SPECIFIC
```

Comment:
MetaDataSymbol is not a class of its own in OMR anymore.  it is a flagged instantiation of RegisterMappedSymbol which is not extended in tr.open.  There is only one CurrentThreadSymbolReference though ... if I am going to test  symRef == SymRefTab->element(currentThreadSymbol), I can get away without J9_SPECIFIC.  Of course, this assumed currentThreadSymbol metaData is immutable in other runtimes as well (although I didn't see this symbol is used in other runtimes).  @0xdaryl  @vijaysun-omr